layouts/grants-and-fellowships/projects/component.jsx

```diff
@@ -72,14 +112,46 @@ const GrantsProjectsSection = ({
     [categories, category]
   );
 
-  useEffect(() => setCountry(countryIso), [countryIso, setCountry]);
+  useEffect(() => setCountry(countryIso), [countryIso]);
 
   useEffect(() => {
-    if (!categories.map(({ label }) => label).includes(category)) {
+    if (!categories?.map(({ label }) => label).includes(category)) {
       setCategory('All');
     }
   }, [country]);
 
+  useEffect(() => {
+    const { country: countrySelected } = query;
+    const isCountrySelectedOnFirstLoad = !!countrySelected;
+
+    if (isCountrySelectedOnFirstLoad) {
+      getMoreProjects({
+        search: countrySelected,
+        per_page: 100,
```

Comment:
wonder if it would be useful to move all these page values into a `const` given that they are being used several times in this file, i. e.

```
const ITEMS_PER_PAGE = Object.freeze({
  CUSTOM: 21,
  DEFAULT: 100,
});
```

and then in this line `per_page: ITEMS_PER_PAGE.DEFAULT`